config/config_helpers.go

```diff
@@ -142,13 +142,21 @@ func CreateTerragruntEvalContext(
 		"get_terragrunt_source_cli_flag":               wrapVoidToStringAsFuncImpl(getTerragruntSourceCliFlag, extensions.TrackInclude, terragruntOptions),
 	}
 
+	terraformCompatibilityFunctions := map[string]function.Function{
```

Comment:
NIT: Why did we need to add these? Add a comment with a link that provides the relevant context.